modules/openapi-generator/src/main/resources/python/rest.mustache

```diff
@@ -168,7 +168,7 @@ class RESTClientObject:
                 ):
                     request_body = None
                     if body is not None:
-                        request_body = json.dumps(body)
+                        request_body = json.dumps(body, ensure_ascii={{ensureAscii}})
```

Comment:
thanks for the PR.

what if `ensureAscii` is not provided in additional properties?